packages/react/src/components/TopBar/topBarMenu.test.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { MenuBar, TopBarMenu, TopBar, MenuItem, OptionsMenuList } from '../..';
+import axe from '../../axe';
+
+const [right, left, down] = [39, 37, 40];
+
+const defaultProps = {
+  id: 'foo'
+};
+
+const optionsMenu = (
+  <OptionsMenuList>
+    <li>option 1</li>
+    <li>option 2</li>
+  </OptionsMenuList>
+);
+
+test('should render children', () => {
+  render(<TopBarMenu {...defaultProps}>{optionsMenu}</TopBarMenu>);
+
+  expect(screen.getAllByRole('menuitem').length).toBeTruthy();
+});
+
+test('should pass-through props', () => {
+  render(
+    <TopBarMenu {...defaultProps} foo="bar" data-testid="TopBarMenu">
```

Comment:
The linter is complaining about the `foo` property since it doesn't exist. Would you be able to change this to something that does exist on the TopBarMenu component?